### ISSUE TITLE
writting local file from oracle doesn't workj

### ISSUE THREAD

**Issue by dridk** [slingdata-io/sling-cli#384]
## Issue Description

Oracle connection works. But I cannot use sling to get output data in local file storage or s3 storage. it hangs during writing. 

- Sling version (`sling --version`): 1.2.19  on red hat 9

- Operating System (`linux`, `mac`, `windows`):  Linux

- Replication Configuration: 

`./sling run --src-conn ORACLE --src-stream 'APPS.ACTES' --tgt-object 'file://test.csv' -d
`

- Log Output (please run command with `-d`): 

```
2024-09-24 19:47:06 DBG Sling version: 1.2.19 (linux amd64)
2024-09-24 19:47:06 DBG type is db-file
2024-09-24 19:47:06 DBG using: {"columns":null,"mode":"incremental","transforms":null}
2024-09-24 19:47:06 DBG using source options: {"empty_as_null":false,"null_if":"NULL","datetime_format":"AUTO","max_decimals":-1}
2024-09-24 19:47:06 DBG using target options: {"header":true,"compression":"auto","concurrency":7,"datetime_format":"auto","delimiter":",","file_max_rows":0,"file_max_bytes":0,"max_decimals":-1,"use_bulk":true,"add_new_columns":true,"column_casing":"source"}
2024-09-24 19:47:06 DBG opened "oracle" connection (conn-oracle-I9Z)
2024-09-24 19:47:06 INF connecting to source database (oracle)
2024-09-24 19:47:06 INF reading from source database
2024-09-24 19:47:06 DBG select * from "APPS"."ACTES"
2024-09-24 19:47:06 INF writing to target file system (file)
2024-09-24 19:47:06 DBG opened "file" connection (conn-file-2er)
2024-09-24 19:47:06 DBG writing to file://bob2.csv [fileRowLimit=0 fileBytesLimit=0 compression=auto concurrency=7 useBufferedStream=false fileFormat=csv singleFile=true]

```

**Comment by flarco** [comment]
Thank you for reporting. I was able to reproduce and fix.
Just released 1.2.20.

**Comment by dridk** [comment]
It works ! Thanks !
